builders/interoperability/xcm/xc20/overview.md

```diff
@@ -7,44 +7,73 @@ description: Learn about the types of cross-chain assets on Moonbeam, in particu
+Within Moonbeam, local XC-20s can only be transferred via their standard ERC-20 interface. By contrast, external XC-20s support both the Substrate and ERC-20 interfaces. Remember that transfers of external XC-20s through the Substrate API will not appear in EVM-based explorers; only transactions performed via the Ethereum API are visible in those explorers.
 
-The main difference between these two types of assets is that local XC-20s are EVM ERC-20s that have XCM capabilities, while external XC-20s are Substrate assets with an ERC-20 interface on top.
+The key difference is that local XC-20s are EVM-native ERC-20s with XCM capabilities, whereas external XC-20s are inherently Substrate assets that expose an ERC-20 interface.
 
 Cross-chain transfers of XC-20s are done using the Polkadot XCM Pallet. To learn how to use this pallet to transfer XC-20s, you can refer to the [Using the Polkadot XCM Pallet](/builders/interoperability/xcm/xc20/send-xc20s/xcm-pallet/){target=\_blank} guide.
 
+## Asset Reserves 
+
+### Remote Reserve Assets
+
+A remote reserve asset is a token whose canonical ledger—the source of truth for minting and burning—resides on a chain different from where it’s currently in use. In the case of xcDOT on Moonbeam, the underlying DOT tokens representing the xcDOT remain locked in Moonbeam’s sovereign account on the Polkadot relay chain, while xcDOT functions as a wrapped representation in Moonbeam’s EVM environment.
+
+Users can hold and transact with xcDOT on Moonbeam (for DeFi, governance, and more), knowing that the underlying DOT is safely locked on the relay chain. At any point, the wrapped xcDOT can be redeemed for the original DOT, effectively burning the xcDOT and unlocking the corresponding DOT tokens on Polkadot.
+
+### Local Reserve Assets 
+
+A local reserve asset on Moonbeam is a token whose canonical ledger—from an XCM perspective—resides natively on Moonbeam. In other words, Moonbeam is the asset’s home chain, where minting and burning take place. 
+
+For example, Wormhole-wrapped ETH (wETH) is considered a local reserve asset on Moonbeam, even though Ethereum is the ultimate source of ETH. Once ETH is wrapped by Wormhole and enters the Polkadot ecosystem via Moonbeam, wETH can be transferred to other parachains through [Moonbeam Routed Liquidity (MRL)](/builders/interoperability/mrl/){target=\_blank}.
+
+The important caveat is that, on a purely Ethereum-level view, ETH remains governed by and minted on Ethereum. However, from an XCM standpoint, wETH on Moonbeam is treated as a local reserve asset, meaning the canonical supply of wETH (as far as Polkadot ecosystems are concerned) exists on Moonbeam.
+
+## Multilocation Overview
```

Comment:
I'm not sure about having this section here if we have a whole page dedicated to it 🤷‍♀️